src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -347,7 +347,7 @@ $externalMargin: 8px;
 	width: var(--header-height);
 	height: var(--header-height);
 
-	&__trigger {
+	#{&}__trigger {
 		width: var(--header-height);
```

Comment:
```suggestion
		width: var(--header-height) !important;
```
Tested and the increased specificity + an extra !important seems necessary otherwise NcButton !important still takes precedence